packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -297,7 +297,7 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     } else {
       final ProcessResult result = processManager.runSync(<String>[javaExecutable, '-version']);
       if (result.exitCode == 0) {
-        final List<String> versionLines = result.stderr.split('\n');
+        final List<String> versionLines = result.stderr.split('\n') as List<String>;
```

Comment:
i'm surprised by this one

Reply:
All of the File/Process APIs are typed as dynamic :(